src/components/AllCitiesOverviewContainer/AllCitiesOverview/AllCitiesOverview.scss

```diff
@@ -156,15 +151,17 @@ $minister-image-width: 214px;
 .subInfoTitle {
   margin-bottom: 32px;
   font-size: 44px;
+  font-weight: 300;
   line-height: 1.2;
-  letter-spacing: -1.8px;
+  letter-spacing: 0.5px;
 }
 
 .subInfoBodyText,
 .infoBodyText {
-  font-size: 22px;
+  font-size: 20px;
+  font-weight: 300;
   line-height: 37px;
-  letter-spacing: -0.7px;
+  letter-spacing: 1px;
```

Comment:
The letters look way too widely spaced to me. 

![image](https://user-images.githubusercontent.com/165087/32420873-c4e73f5a-c2e5-11e7-920b-282f7059ba81.png)

Reply:
What browser is that? It doesn't look right.

Here's how it looks for me, Chrome on Mac and Edge and Chrome on Windows all looked fine. The designs are overlaid (the lighter text)

![image](https://user-images.githubusercontent.com/4443482/32421982-58c99f46-c2f1-11e7-8664-5278015cb4f9.png)

Reply:
Chrome 61 on Mac.

Reply:
Can you screenshot the whole screen? This is Chrome 61 macOS for me:
![image](https://user-images.githubusercontent.com/4443482/32423629-8682c908-c2fb-11e7-842c-3efe57bc199a.png)